pkg/mev/gas_bundle_estimator.go

```diff
@@ -0,0 +1,50 @@
+package mev
+
+import (
+	"context"
+
+	"github.com/ethereum/go-ethereum"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/common/hexutil"
+	"github.com/ethereum/go-ethereum/ethclient"
+	"github.com/ethereum/go-ethereum/ethclient/gethclient"
+)
+
+type GasBundleEstimator struct {
+	ethClient *ethclient.Client
```

Comment:
Should be `rpc.Client` instead of `ethclient.Client`

Reply:
fixed